details.html

```diff
@@ -132,6 +132,108 @@ <h3 ng-if="isOthers" class="started-header"> Other details </h3>
               <div class="get-started-body others-md">
               </div>
             </div>
+            <div class="get-started additional-information">
+              <hr>
+              <div class="get-stared-header">
+                <h3 class="started-header"> Additional Information </h3>
+              </div>
+              <div class="get-started-body row">
+                <div ng-if="appData.updated !== undefined && appData.updated !== ''" class="col-md-4 add-info">
+                  <div class="info-type">
+                    <h5 class="info-header">
+                      <strong>Updated</strong>
+                    </h5>
+                  </div>
+                  <div class="info-body">
+                    {{appData.updated}}
+                  </div>
+                </div>
+                <div ng-if="appData.version !== undefined && appData.version !== ''" class="col-md-4 add-info">
+                  <div class="info-type">
+                    <h5 class="info-header">
+                      <strong>Version</strong>
+                    </h5>
+                  </div>
+                  <div class="info-body">
+                    {{appData.version}}
+                  </div>
+                </div>
+                <div ng-if="appData.appSource !== undefined && appData.appSource !== ''" class="col-md-4 add-info">
+                  <div class="info-type">
+                    <h5 class="info-header">
+                      <strong>App Source</strong>
+                    </h5>
+                  </div>
+                  <div class="info-body">
+                    <a href="{{appData.appSource}}"> View Source </a>
+                  </div>
+                </div>
+                <div ng-if="(appData.author.url !== undefined && appData.author.url !== '') ||
+                  (appData.author.email !== undefined && appData.author.email !== '')" class="col-md-4 add-info">
+                  <div class="info-type">
+                    <h5 class="info-header">
+                      <strong>Developer</strong>
+                    </h5>
+                  </div>
+                  <div class="info-body">
+                    <span ng-if="appData.author.url !== undefined && appData.author.url !== ''">
+                      <a href="{{appData.author.url}}"> Vist developer's website </a>
+                    </span><br>
+                    <span ng-if="appData.author.email !== undefined && appData.author.email !== ''">
+                      <a href="mailto:{{appData.author.email}}"> Send an email </a>
```

Comment:
Thats strange! Basically mailto is supposed to open the default mail client on that platform, for example on android phones it would open gmail app or provide a list of mail clients to choose for, or on linux it might open thunderbird or any other mail client as configured. Ideally it should not do anything when nothing is installed. To what page is it redirecting you? What is the url?